dotCMS/src/integration-test/java/com/dotcms/saml/SamlConfigurationServiceTest.java

```diff
@@ -0,0 +1,34 @@
+package com.dotcms.saml;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.util.Config;
+import org.apache.felix.framework.OSGIUtil;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SamlConfigurationServiceTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+
+        if (!OSGIUtil.getInstance().isInitialized()) {
+            OSGIUtil.getInstance().initializeFramework(Config.CONTEXT);
+        }
+    }
+
+    @Test
+    public void test_getConfigAsString() {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5956529)